### PR TITLE
chore: fix linting 

### PR DESCRIPTION
**Pull request by Shahroz16**
fixes all the linting issues except from `fixme` and `todo`

### CODE REVIEW

File: Tests/Tracking/migration/DataPipelineMigrationAssistantTests.swift [hunk 1]
```diff
@@ -31,16 +31,28 @@ class DataPipelineMigrationAssistantTests: UnitTest {
...
+        guard let fileManagerQueueStorage = queueStorage as? FileManagerQueueStorage else {
+            XCTFail("queueStorage could not be cast to FileManagerQueueStorage")
+            return
+        }
+
         let counter = 3000
         for _ in 1 ... counter {
-            let givenCreatedTask = (queueStorage as! FileManagerQueueStorage).create(type: givenType.rawValue, data: givenQueueTaskData, groupStart: nil, blockingGroups: nil)
-                .createdTask!
+            guard let givenCreatedTask = fileManagerQueueStorage.create(type: givenType.rawValue, data: givenQueueTaskData, groupStart: nil, blockingGroups: nil).createdTask else {
```
Comment (Shahroz16) [inline Tests/Tracking/migration/DataPipelineMigrationAssistantTests.swift:47]: fixes forced casting

File: Sources/Common/Communication/EventBus.swift [hunk 2]
```diff
@@ -57,12 +57,14 @@ class EventBusObserversHolder {
...
 }
 
+// swiftlint:disable orphaned_doc_comment
 /// A shared implementation of `EventBus` using an actor model for thread-safe operations.
 /// This actor manages the distribution of events to registered observers and uses
 /// `NotificationCenter` for event delivery. It ensures that event handling is thread-safe
 /// and observers are managed efficiently.
 // sourcery: InjectRegisterShared = "EventBus"
 // sourcery: InjectSingleton
+// swiftlint:enable orphaned_doc_comment
```
Comment (mrehan27) [inline Sources/Common/Communication/EventBus.swift:67]: Wondering why it was orphan comment when it is exactly above the class 😅

File: Sources/Common/Communication/EventStorage.swift [hunk 3]
```diff
@@ -31,11 +31,13 @@ enum EventBusError: Error {
     case decodingError
 }
 
+// swiftlint:disable orphaned_doc_comment
 /// An actor that manages event storage, providing thread-safe operations for storing, loading, and removing events.
 ///
 /// This class handles the persistence of events using the file system, ensuring data integrity and consistency.
 // sourcery: InjectRegisterShared = "EventStorage"
 // sourcery: InjectSingleton
+// swiftlint:enable orphaned_doc_comment
```
Comment (Shahroz16) [inline Sources/Common/Communication/EventStorage.swift:40]: `orphaned_doc_comment`

expects doc comment should be attached to a declaration

File: Sources/DataPipeline/DataPipelineConfigOptions.swift [hunk 4]
```diff
@@ -14,7 +14,6 @@ import Segment
  dataPipelineConfig.autoTrackScreenViews = false
  ```
  */
-// TODO: [CDP] Update docs to after module initialization changes for clarity on its usage
```
Comment (mrehan27) [inline Sources/DataPipeline/DataPipelineConfigOptions.swift:17]: Did we complete this TODO? 🤔

File: Sources/MessagingPush/MessagingPushConfigOptions.swift [hunk 5]
```diff
@@ -9,7 +9,6 @@
  pushMessagingConfig.autoTrackPushEvents = false
  ```
  */
-// TODO: [CDP] Update docs to after module initialization changes for clarity on its usage
```
Comment (mrehan27) [inline Sources/MessagingPush/MessagingPushConfigOptions.swift:12]: Did we complete this TODO too? 🤔

**Review by mrehan27 — APPROVED** [review]
A few questions, looks good to me 👍🏻

**Review by ami-aman — APPROVED** [review]

File: Sources/Common/Communication/EventBus.swift (hunk 2 above)
Comment (Shahroz16) [reply Sources/Common/Communication/EventBus.swift:67]: It means we should continue using `///` till the document start and not add `//` for sourcery.

File: Sources/MessagingPush/MessagingPushConfigOptions.swift (hunk 5 above)
Comment (Shahroz16) [reply Sources/MessagingPush/MessagingPushConfigOptions.swift:12]: We have a separate ticket for docs, ill add this in it.

**Pull request merged**